plugins/modules/net_tools/nmcli.py

```diff
@@ -314,15 +314,23 @@
        type: str
     ip_tunnel_dev:
         description:
-            - This is used with IPIP/SIT - parent device this IPIP/SIT tunnel, can use ifname.
+            - This is used with GRE/IPIP/SIT - parent device this GRE/IPIP/SIT tunnel, can use ifname.
         type: str
     ip_tunnel_remote:
        description:
-            - This is used with IPIP/SIT - IPIP/SIT destination IP address.
+            - This is used with GRE/IPIP/SIT - GRE/IPIP/SIT destination IP address.
        type: str
     ip_tunnel_local:
        description:
-            - This is used with IPIP/SIT - IPIP/SIT local IP address.
+            - This is used with GRE/IPIP/SIT - GRE/IPIP/SIT local IP address.
+       type: str
+    ip_tunnel_input_key:
+       description:
+            - This is used with GRE - The key used for tunnel input packets
+       type: str
```

Comment:
```suggestion
       type: str
       version_added: 3.6.0
```
Same for the other new option.